source/core/src/main/com/csse3200/game/areas/EnemyRoom.java

```diff
@@ -111,7 +131,7 @@ public void makeAllAnimalDead() {
         for (Entity entity : enemies) {
             CombatStatsComponent combatStatsComponent = entity.getComponent(CombatStatsComponent.class);
             if (combatStatsComponent != null) {
-                combatStatsComponent.setHealth(1);
+                combatStatsComponent.setHealth(0);
```

Comment:
Health needs to go to 1 before being hit again to be correct

Reply:
Fixed